package.json

```diff
@@ -8,7 +8,7 @@
     "classnames": "2.1.2",
     "flux": "^2.0.3",
     "json-stable-stringify": "1.0.0",
-    "keen-analysis": "^1.1.0",
+    "keen-analysis": "1.1.0",
```

Comment:
@sfalkoff His PR should overwrite my pinning, so I'm gonna leave this in (that will avoid CI/build errors depending on which merges first)